src/Pages/Join/Join.scss

```
@@ -0,0 +1,239 @@
+@import "./../../Styles/common.scss";
+
+* {
+  box-sizing: border-box;
+  font-family: HelveticaNeueW01-75Bold;
+}
+
+.error {
+  border: 1px solid red;
+}
+
+.Join {
+  .teaSoMain {
+    position: relative;
+    .teaSoLogoBox {
+      position: absolute;
```

```suggestion
  .teaSoMain {
    position: relative;
    
    .teaSoLogoBox {
      position: absolute;
```
새로운 셀렉터 나올 때는 한줄 띄워주세요